src/main/java/frc/robot/commands/BangBangShooter.java

```diff
@@ -155,7 +158,7 @@ else if (currentControlMode == ControlMethod.HOLD) {
   @Override
   public void end(boolean interrupted) {
     shooter.stopAll();
-    shooter.setLEDRing(false);
+    shooter.vision.setLedRing(false);
```

Comment:
Shouldn't we just do shooter.vision.setMode("intake") here? I don't think we want to leave it in goalfinder mode with the LED turned off.

Reply:
Yes, use shooter.vision.setMode("intake")